ambari-server/src/main/resources/stacks/HDP/2.6/upgrades/config-upgrade.xml

```diff
@@ -0,0 +1,778 @@
+<?xml version="1.0"?>
```

Comment:
Did you mean to add this file (and others) here?  I thought these were in a different repo now.

Reply:
EWWWWW! Nice catch. Let me fix it.